packages/client/src/ext/jwt-simple.ts

```diff
@@ -0,0 +1,213 @@
+/*
+ * jwt-simple
+ *
+ * JSON Web Token encode and decode module for node.js
+ *
+ * Copyright(c) 2011 Kazuhito Hokamura
+ * MIT Licensed
+ */
+
+/**
+ * module dependencies
+ */
+import { base64 } from '@scure/base'
+import crypto from 'crypto'
+
+/**
+ * support algorithm mapping
+ */
+export type TAlgorithm = 'HS256' | 'HS384' | 'HS512' | 'RS256'
+const algorithmMap: Record<string, string> = {
+  HS256: 'sha256',
+  HS384: 'sha384',
+  HS512: 'sha512',
+  RS256: 'RSA-SHA256',
+}
+
+/**
+ * Map algorithm to hmac or sign type, to determine which crypto function to use
+ */
+const typeMap: Record<string, string> = {
+  HS256: 'hmac',
+  HS384: 'hmac',
+  HS512: 'hmac',
+  RS256: 'sign',
+}
+
+/**
+ * expose object
+ */
+export const jwt = module.exports
+
+/**
+ * version
+ */
+jwt.version = '0.5.6'
```

Comment:
We don't need this anymore since it's going to be part of our code base now.

Reply:
done